cmd/agentctl/Dockerfile.windows

```diff
@@ -0,0 +1,15 @@
+FROM grafana/agent-build-image:latest-windows as builder
```

Comment:
We should use the explicit version here instead of `latest` for reproducible builds (also we don't push a `latest` tag anyway)